### ISSUE TITLE
Creating a new list and adding it to the root list

### ISSUE THREAD

**Issue by 0xferit** [kleros/gtcr#231]
The user interface does not guide the user to submit the newly created list into the root list. This is very bad for the user experience because, after deployment, the user has to find the contract address of the newly deployed list, go back to the root list, click submit, enter this address of the newly deployed list and execute the transaction. This is very cryptic and confusing. An unsuspecting user would be searching for the newly deployed list, unable to find it, thinking that the software is buggy and would leave.

In conclusion, we need to guide users on UI and help them on adding their list to the root list.

**Comment by 0xferit** [comment]
I followed the flow again, and now I noticed that there is an infobox to inform users about this. So it's not totally true that UI is not guiding at all, but the guidance is inadequate. I missed it and probably many others will.

I'm going to lower the priority to medium.
